libsodium-bindings/src/LibSodium/Bindings/AEAD.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE CApiFFI #-}
+
+module LibSodium.Bindings.AEAD where
```

Comment:
The haddock front-matter is missing, I recommend you just adapt one from the other modules. :)